packages/react-components/src/echarts/unboundedZoom/constants.ts

```diff
@@ -32,15 +44,20 @@ export const DEFAULT_X_AXIS: XAXisComponentOption = {
   max: 4102513200000, // Jan 01 2100 19:00:00 UTC
 };
 
-export const DEFAULT_TOOLBOX = {
+export const DEFAULT_TOOLBOX_GESTURES_ENABLED: ToolboxComponentOption = {
   show: true,
   top: 0,
   right: 8,
   feature: {
     dataZoom: {
       title: { back: 'Undo\nzoom' },
       filterMode: 'none',
+      xAxisIndex: 'all',
```

Comment:
Curious what this does? allow events to trigger off any x axis in the chart instance?

Reply:
not needed, it was resetting it back to the default value.
> Defines which [xAxis](https://echarts.apache.org/en/option.html#xAxis) should be controlled. By default, it controls all x axes.